judoscale-solid_queue/test/test_helper.rb

```diff
@@ -41,10 +44,15 @@ class TestRailsApp < Rails::Application
 Minitest.after_run {
   ActiveRecord::Tasks::DatabaseTasks.drop(DATABASE_URL)
 }
-ActiveRecord::Base.establish_connection(DATABASE_URL)
+ActiveRecord::Base.configurations = {test: {url: DATABASE_URL}}
+ActiveRecord::Base.establish_connection(:test)
```

Comment:
Need to establish the connection via the current Rails.env to `load_schema_current` below.